README.md

```diff
@@ -0,0 +1,38 @@
+# OVGT - Open Source VGT Controller
+
+An open-source Variable Geometry Turbo controller for Teensy 4.1 and MicroMod Teensy platforms.
+
+## Hardware Requirements
+
+### Supported Microcontrollers
+- Teensy 4.1 (primary development target)
+- MicroMod Teensy (secondary target)
+
+### Required Components
```

Comment:
Can we call out that the LCD is not a required, but is optional and mainly used for debugging?